### PR TITLE
fix: adds new line after comment

### PR DESCRIPTION
**Pull request by madou**

### CODE REVIEW

File: packages/babel-plugin/src/__tests__/index.test.tsx [hunk 1]
```diff
@@ -8,6 +8,19 @@ const babelOpts: TransformOptions = {
 };
 
 describe('babel plugin', () => {
+  it('should not comment file if not transformation occured', () => {
```
Comment (pgmanutd) [inline packages/babel-plugin/src/__tests__/index.test.tsx:11]: nit:
```suggestion
  it('should not comment file if no transformation occurred', () => {
```

File: packages/babel-plugin/src/index.tsx [hunk 2]
```diff
@@ -50,6 +50,9 @@ export default declare<State>((api) => {
             'leading',
             ` ${parseFilename(state.filename)} generated by ${pkgJson.name} v${version} `
           );
+
+          // Add a line break the comment
+          path.unshiftContainer('body', t.noop());
```
Comment (pgmanutd) [inline packages/babel-plugin/src/index.tsx:55]: oooo `babel` utilities `t.noop`. Nice.

**Review by pgmanutd — APPROVED** [review]
lgtm

File: packages/babel-plugin/src/index.tsx (hunk 2 above)
Comment (madou) [reply packages/babel-plugin/src/index.tsx:55]: yeah you find wild things going through the api.. not documented anywhere of course

**Pull request merged**